### PR TITLE
make a copy task

### PR DESCRIPTION
**Pull request by daveol**
make a copy task in grunt for easier debugging of rdm.js

### CODE REVIEW

**Comment by nomis52** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -48,4 +48,22 @@ or if you are continuously adjusting the source and testing it,
 and you just want to automate the compressing of rdm.js run
 ```bash
 grunt compress-watch
-```
\ No newline at end of file
+```
+
+## Debug rdm.js
+
+if you are having troubles with debugging the minified source of rdm.js you can copy
```
Comment (peternewman) [inline README.md]: SPaG If

Comment (peternewman) [inline README.md]: SPaG trouble

File: README.md [hunk 2]
```diff
@@ -48,4 +48,22 @@ or if you are continuously adjusting the source and testing it,
...
+the files to the static dir using
+```bash
+grunt copy-once
+```
+or if you want a automatic copy run
```
Comment (peternewman) [inline README.md]: SPaG an

File: README.md [hunk 3]
```diff
@@ -48,4 +48,22 @@ or if you are continuously adjusting the source and testing it,
...
+```bash
+grunt copy-watch
+```
+### Important!
+when you are done debugging don't forget to run
```
Comment (peternewman) [inline README.md]: SPaG When

**Pull request merged**